ZUGFeRD/InvoiceDescriptor1Writer.cs

```diff
@@ -73,6 +73,14 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream)
             Writer.WriteStartElement("ram:TestIndicator");
             Writer.WriteElementString("udt:Indicator", this.Descriptor.IsTest ? "true" : "false");
             Writer.WriteEndElement(); // !ram:TestIndicator
+
+            if (Descriptor.Profile == Profile.Extended && !String.IsNullOrEmpty(this.Descriptor.BusinessProcess)) 

```

Comment:
can you please use the profile parameter of WriteStartElement() instead of checking the profile explicitly in the if () clause?

Reply:
Indeed I had not seen the optional parameter, I modified.